WordPress/src/main/res/values/strings.xml

```diff
@@ -2306,45 +2295,6 @@
 
 
     <!-- Site Creation -->
```

Comment:
I think we should remove this comment as well. Otherwise, it looks as if the quick start strings are `Site Creation` strings.

Reply:
Addressed in 28c51d42878a11e6e55e183c39e8f57a51bef21e as part of #9053.

Reply:
Oh, I missed that. Thanks!